### PR TITLE
feat: Send order via post to unrepresented respondents (FPLA-2432)

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
https://tools.hmcts.net/jira/browse/FPLA-2432

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/SendDocumentController.java [hunk 1]
```diff
@@ -14,18 +14,24 @@
...
+import java.util.ArrayList;
 import java.util.List;
 
 import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.POST;
 
+/*
+@deprecated - use SendDocumentService instead
+*/
+
+@Deprecated
```
Comment (joez-f) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/SendDocumentController.java:34]: why deprecate instead of remove? just curious

Comment (joez-f) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/SendDocumentController.java:34]: ah I see, existing places have not been changed for now 👍 e.g CMO, NoH

Comment (joez-f) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/SendDocumentController.java:34]: should we make tech debt ticket to update existing code?

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerSubmittedTest.java [hunk 2]
```diff
@@ -1,109 +1,199 @@
...
+
+    private static CaseData CASE_DATA = CaseData.builder()
+        .id(CASE_ID)
+        .caseLocalAuthority(LOCAL_AUTHORITY_1_CODE)
+        .familyManCaseNumber(FAMILY_MAN_CASE_NUMBER)
+        .orderCollection(createOrders(ORDER))
+        .respondents1(wrapElements(RESPONDENT_NOT_REPRESENTED, RESPONDENT_WITHOUT_ADDRESS, RESPONDENT_REPRESENTED))
+        .representatives(List.of(REPRESENTATIVE_POST, REPRESENTATIVE_DIGITAL, REPRESENTATIVE_EMAIL))
+        .build();
+
```
Comment (imran-hmcts) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerSubmittedTest.java:162]: Minor thing: Above fields could be declared as final for consistency.

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/SendDocumentController.java (hunk 1 above)
Comment (tomaszpowroznik) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/SendDocumentController.java:34]: will do cleanup at the end of series of stories

**Review by imran-hmcts — APPROVED** [review]

**Pull request merged**